src/plugin.js

```diff
@@ -232,8 +232,10 @@ Chart.plugins.register({
 			update = updates[i];
 			if (update[1]) {
 				label = update[0][EXPANDO_KEY];
-				label.$context.active = (update[1] === 1);
-				label.update(label.$context);
+				if (label && label.$context) {
```

Comment:
I would only test for `label` since `label.$context` should always be defined, else that's a bug and should not be silently ignored.